lib/godfist/league_rates.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Godfist.LeagueRates do
+  @moduledoc false
+
+  # Handles checking the information passed and assigning the correct
+  # limit to the request.
+
+  use GenServer
+
+  alias Godfist.HTTP
+
+  # Rates for different servers.
+  @rates [
+    # "League" endpoints/servers
```

Comment:
It sounds like from the docs that these are the default values but they can be overriden on a per API key basis. So it might be good to have a way to easily override or configure these.

Reply:
Hmm, there are a lot of limits, I could make them overridable but that would be a lot of options. 🤔